tests/markFeatureWriter_test.py

```diff
@@ -16,9 +16,10 @@ def test_add_classes(self):
         glyph.appendAnchor(glyph.anchorClass(
             anchorDict={'name': '_bottom', 'x': 100, 'y': 0}))
         lines = []
-        writer = MarkFeatureWriter(
-            ufo, anchorList=(('bottom', '_bottom'),), mkmkAnchorList=(),
-            ligaAnchorList=((('top_1', 'top_2'), '_top'),))
+        writer = MarkFeatureWriter(ufo)
+        writer.anchorList = (('bottom', '_bottom'),)
+        writer.mkmkAnchorList = ()
+        writer.ligaAnchorList = ((('top_1', 'top_2'), '_top'),)
```

Comment:
Point is you don't always want to run the inference, the class should make it easy not to. See 422d5852c932b73e62b3930ce8106680890b70f8 for a corresponding change.